packages/react-scripts/package.json

```diff
@@ -87,6 +87,7 @@
     "webpack-bundle-analyzer": "3.9.0",
     "webpack-dev-server": "3.11.1",
     "webpack-manifest-plugin": "2.2.0",
+    "webpack-subresource-integrity": "1.5.2",
```

Comment:
The latest [version](https://www.npmjs.com/package/webpack-subresource-integrity) of this package is `5.0.0`. It's the next version after this `1.5.2`, for some reason. 🤔 

Is there a reason that we are targeting the older package? Maybe the `5.0.0` version doesn't work with `react-scripts`'s version of webpack?

Reply:
The reason we are using 1.5.2 is because we are using webpack 4.